graph-deps

```diff
@@ -0,0 +1,223 @@
+#!/usr/bin/env python
+#
+# This script takes a dmd dependency file, and outputs a graph in graphviz
+# format also, detect cycles
+#
+# Based on: https://github.com/dellort/lumbricus/blob/d2/src/scripts/deps.py
+# Which used code from: http://www.logarithmic.net/pfh/blog/01208083168
```

Comment:
It was on purpose, they are like independent items. If you think is confusing I can change it though, but I will only after getting a confirmation.